libraries/AP_BoardConfig/board_drivers.cpp

```diff
@@ -152,6 +152,90 @@ bool AP_BoardConfig::spi_check_register(const char *devname, uint8_t regnum, uin
     return v == value;
 }
 
+/**
+ * MS56XX crc4 method from datasheet for 16 bytes (8 short values)
+ */
+static uint16_t crc4(uint16_t *data)
```

Comment:
I'd rather make the functions in AP_Baro_MS5611 be public static and call them, or move the crc function to AP_Math/crc.c with suitable name